devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -70,6 +87,62 @@ def parse(String description) {
 	result
 }
 
+// cmd.endPoints includes the USB ports but we don't want to expose them as child devices since they cannot be controlled so hardcode to just include the outlets
+def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelEndPointReport cmd, ep = null) {
+	if(!childDevices) {
+		if (isZoozZenStripV2()) {
+			addChildSwitches(5)
+		} else if (isZoozDoublePlug()) {
+			addChildSwitches(2)
+		} else {
+			addChildSwitches(cmd.endPoints)
+		}
+	}
+	response([
+			resetAll(),
```

Comment:
The spacing looks weird and it looks weird in other DTHs, @greens what am I missing?

Reply:
Not how I would do it, but they're double-tabbing because of paren+bracket

Reply:
That snippet doesn't look that bad in the "intelliJ Idea",  should i fix it ?